backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeService.java

```diff
@@ -55,6 +56,22 @@ public DeviceType getDeviceType(String internalId) {
         .orElseThrow(() -> new IllegalGraphqlArgumentException("invalid device type ID"));
   }
 
+  public List<DeviceSpecimenType> getDeviceSpecimenTypesByIds(List<String> deviceSpecimenTypeIds) {
```

Comment:
**suggestion:** This `List<String>` should probably be `List<UUID>`

Reply:
Follow-up on this: The GraphQL schema has these as `List<String>` in the mutation. We should move away from this, but let's fix it here (and everywhere else) _downstream_ of the mutation for now, and create a followup ticket to fix the schema

Reply:
See https://github.com/CDCgov/prime-simplereport/pull/2687/files/4faa24fa1b8bc8cbd2e3c78610a3f4b34101f91b#r723618003.